spec/models/profile_spec.rb

```diff
@@ -379,14 +643,15 @@
       profile.activate_after_password_reset
 
       expect(profile.activated_at).to be_present
-      expect(profile.active).to eq true # changed
+      expect(profile.active).to eq(true) # changed
       expect(profile.deactivation_reason).to be_nil # changed
       expect(profile.fraud_review_pending?).to eq(false)
       expect(profile.gpo_verification_pending_at).to be_nil
       expect(profile.initiating_service_provider).to be_nil
       expect(profile.verified_at).to eq verified_at
     end
 
+    # ??? This method still nils out deactivation_reason even though it raises
     it 'does not activate a profile if it has a pending reason' do
```

Comment:
Test title could be more descriptive.
```suggestion
    it 'does not activate a profile after password reset if it has fraud review pending' do
```

Maybe it's removing password_reset as the deactivation reason because they used their Personal Key to recover the PII bundle? Do they have a Personal Key if they're fraud review pending? Maybe these password reset specs are left over from when the user saw the Personal Key before entering their GPO code.